ext/test.c

```diff
@@ -216,6 +216,10 @@ zend_class_entry *test_unsettest_ce;
 zend_class_entry *test_usetest_ce;
 zend_class_entry *test_vars_ce;
 
+#ifdef ZTS
+ZEND_TSRMLS_CACHE_DEFINE()
+#endif
```

Comment:
@remicollet 
Shouldn't it be changed as follows?

```c
/* implement standard "stub" routine to introduce ourselves to Zend */
#ifdef COMPILE_DL_%PROJECT_UPPER%
# ifdef ZTS
ZEND_TSRMLS_CACHE_DEFINE()
# endif
ZEND_GET_MODULE(%PROJECT_LOWER_SAFE%)
#endif
```

Reply:
indeed, even if don't see in which case `COMPILE_DL_%PROJECT_UPPER%` won't be define (excepted stupid edge case of people merge some ext in php-src)

Reply:
Thank you